tst/testbugfix/2018-12-17-gquotient.tst

```diff
@@ -0,0 +1,7 @@
+# reported by Giles Gardam. The code for eliminating element orders can run astray if 
+# a quotent by a generator power is cyclic, but also has cyclic subgroups of infinite
+# index.
+gap> F := FreeGroup("a", "b", "c");;
+gap> G := F/ParseRelators(F,"a2b5a2b2C2,a5c3B2");;
+gap> Length(GQuotients(G,AlternatingGroup(5)));
```

Comment:
This tests was running for several minutes on my machine, then run out of memory.

That's not acceptable for tst/testbugfix, whch we run as part of our CI on every pull request. It could be added to `tst/testextra`, though (perhaps in a new dir `tst/testextra/bugfix`)

Reply:
Let me check. This example should run in under a second. Probably it is the assertions again...

Reply:
Dear @fingolfin 
That is really strange. I just rebased the branch, recompiled and ran the file with `Test`, and it took indeed under a second. Also, in the travis log I get

    testing: /home/travis/build/gap-system/gap/tst/testbugfix/2018-12-17-gquotient.tst
     923 ms (24 ms GC) and 28.7MB allocated for 2018-12-17-gquotient.tst

which is the same magnitude.
Is it possible that you are loading something else that interferes? Could it be the Julia issue Thomas was observing?